src/Components/Molecules/gallery-images/gallery-images.js

```diff
@@ -8,7 +8,7 @@ const GalleryImages = () => {
             {
                 GalleryImages.map(post => {
                     return (
-                        <img src={post.path} alt={post.altTag} key={post.altTag}/>
+                        <img src={post.path} alt={post.altTag} key={post.id}/>
```

Comment:
It's not necessary to add ids to json in this case. You could add index to mapping `GalleryImages.map((post, index) => ` and set `key={index}`. This approach could be used in most case. The example of a case where it won't work would be the case where you want to animate elements if you switch their position. If you are using custom ids as keys then all classes will be preserved and as soon as element will get new styles transition will be executed, but if you are using indexes elements which have changed their positions will also have their keys updated which would re-render those and hence no animation will be triggered.